readthedocs/templates/builds/versionautomationrule_list.html

```diff
@@ -0,0 +1,74 @@
+{% extends "projects/project_edit_base.html" %}
+
+{% load i18n %}
+
+{% block title %}{% trans "Automation Rules" %}{% endblock %}
+
+{% block nav-dashboard %} class="active"{% endblock %}
+
+{% block project-automation-rules-active %}active{% endblock %}
+{% block project_edit_content_header %}{% trans "Automation Rules" %}{% endblock %}
+
+{% block project_edit_content %}
+
+  <p class="help_text">
+    {% blocktrans trimmed %}
+      Execute some actions over new versions.
+      Check the <a href="https://docs.readthedocs.io/page/automation-rules.html">documentation</a> for more information.
+    {% endblocktrans %}
+  </p>
+
+
+  <div class="button-bar">
+    <ul>
+      <li>
+        <a class="button"
+           href="{% url 'projects_automation_rule_regex_create' project_slug=project.slug %}">
+          {% trans "Add Rule" %}
+        </a>
+      </li>
+    </ul>
+  </div>
+
+  <div class="module-list">
+    <div class="module-list-wrapper">
+      <ul>
+        {% for rule in object_list %}
+          <li class="module-item">
+            <a href="{{ rule.get_edit_url }}">
+              {{ rule.get_description }}
+            </a>
+            <span class="quiet">
+              ({{ rule.get_version_type_display }})
+            </span>
+            <ul class="module-item-menu">
+              {% if not forloop.first %}
+                <li class="automation-rule">
+                  <form method="post" action="{% url 'projects_automation_rule_move' project_slug=project.slug automation_rule_pk=rule.pk steps=-1 %}">
+                    {% csrf_token %}
+                    <input type="submit" value="&#xf062">
```

Comment:
What's the value here? Is this a font awesome thing? After mentioning a11y change above, it's probably worth mentioning here as well: we might want to start adding more context for screen readers. This could just use an aria-label so the screen reader can read this button: 

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute

Reply:
Yes, it's a font awesome button. Arial label added